tools/shared.py

```diff
@@ -1921,7 +1921,7 @@ def link_lld(args, target, opts=[], lto_level=0):
     #   cmd.append('--strip-debug')
 
     if Settings.EXPORT_ALL:
-      cmd += ['--no-gc-sections', '--export-all']
```

Comment:
This looks fine if it works with the linker, but just curious, is this unrelated, or somehow connected to these changes?

Reply:
Reverted this part